build.sbt

```diff
@@ -348,17 +345,10 @@ lazy val sparkRunnerDependencies = Seq(
   "org.apache.spark" %% "spark-streaming" % sparkVersion
 )
 
-// only available for scala 2.11 & 2.12
-// https://issues.apache.org/jira/browse/FLINK-13414
-// beam-runners-flink-1.14 by default pulls 2.12 scala libs,
-// exclude those libs and add them manually with proper scala version
```

Comment:
`beam-runners-flink-1.15` only depends on java lib now.
`flink-clients` and `flink-streaming-java` are now also java libs